src/app.rs

```diff
@@ -199,7 +184,23 @@ impl AsyncComponent for App {
 							},
 						}
 					}
-				}
+				},
+                gtk::InfoBar {
+                    set_message_type: gtk::MessageType::Warning,
+										set_visible: PROFILE == "Devel",
+										#[watch]
+										set_revealed: model.warning_revealed,
+										set_show_close_button: true,
+										connect_response[sender] => move |_, _| {
+                        sender.input_sender().send(Event::CloseWarning).unwrap()
+                    },
+										gtk4::Label {
+                        set_wrap: true,
+											set_natural_wrap_mode: gtk4::NaturalWrapMode::None,
+											add_css_class: "warning",
+											set_text: fl!("alpha-warning")
+                    }
```

Comment:
The indentation looks odd here.

Reply:
Ugh... `rustfmt` is screwing with the macro

Reply:
Latest commit should fix it